test/blackbox-tests/test-cases/melange/ocaml-flags.t

```diff
@@ -22,12 +22,35 @@ The code in main contains unused var (warning 26) and illegal backlash (warning
   > print_endline "hello"
   > EOF
 
-Building does not fail, warnings are silenced as melange builds reuse ocamlc flags
+Building does not fail, warnings are silenced
 
   $ dune build output/main.js
   $ node _build/default/output/main.js
   hello
 
+Update dune file to use ocamlc_flags
+
+  $ cat > dune <<EOF
+  > (melange.emit
+  >  (target output)
+  >  (entries main)
+  >  (module_system commonjs)
+  >  (ocamlc_flags -w -14-26))
```

Comment:
I don't think we should even accept this field in melange.emit.

Reply:
Fixed in https://github.com/ocaml/dune/pull/6569/commits/8043ed0ee110ff00f3f68a6fbe501505e4a60324.